docs/backends/gcloud.rst

```diff
@@ -136,6 +143,19 @@ Sets Cache-Control HTTP header for the file, more about HTTP caching can be foun
 Subdirectory in which the files will be stored.
 Defaults to the root of the bucket.
 
+``GS_EXPIRES_IN`` (optional: default is ``timedelta(seconds=86400)``)
+
+The time that a generated URL is valid before expiration. The default is 1 day.
+Public files will return a url that does not expire. Files will be signed by
+the credentials provided to django-storages (See GS_CREDENTIALS).
+
+Note: Default Google Compute Engine (GCE) Service accounts are
+`unable to sign urls <https://googlecloudplatform.github.io/google-cloud-python/latest/storage/blobs.html#google.cloud.storage.blob.Blob.generate_signed_url>`_.
+
+The ``GS_EXPIRES_IN`` value is handled by the underlying `Google library  <https://googlecloudplatform.github.io/google-cloud-python/latest/storage/blobs.html#google.cloud.storage.blob.Blob.generate_signed_url>`_.
+It supports `timedelta`, `datetime`, or `integer` seconds since epoch time.
```

Comment:
Kind of:

`timedelta` is added to `.utcnow()` while `date` and `int` are literal times. It would be pretty weird to be using `date` and `int` in this library correctly.

Reply:
Shrug... maybe you want the URL to expire on Christmas in 2020. I expect most people will use timedelta.

Reply:
Yeah but this is where `expires_on` vs `expires_in` comes into play. If I put `timedelta(hours=4)` then that is an `expires_in` scenario. If I put `2020-12-25` then that is `expires_on`. It seems very confusing.

Reply:
That is why I originally had it at expires_on because it expires_on (now + timedelta) or an absolute time.... 

I would also argue this is a very minor point because without this change the current version is broken. 
My guess it 99% of people set a timedelta, but I do not think we should block the underlying Google functionality by disallowing the other options.